earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp

```diff
@@ -216,17 +219,29 @@ QFileDialog* RasterAssetWidget::FileDialog() {
         }
       }
     }
-    // For JPEG2000 and MrSID, check if GDAL is built
-    // with specific format support.
+    // Add file filters:
+    QString supported_files_filter = "Supported images (";
+    if (add_dem_file_filter)
```

Comment:
Its confusing to have DEM for Imagery. We should not check for it and add it here. Its for Terrain only.